test/src/test/java/hudson/PluginManagerTest.java

```diff
@@ -711,4 +735,101 @@ public void installNecessaryPluginsTest() throws Exception {
         // Redirect reported 404 before bug was fixed
         assertEquals("Bad response for installNecessaryPlugins", 200, response.statusCode());
     }
+
+    @Test
+    @Issue("SECURITY-2823")
+    public void verifyUploadedPluginPermission() throws Exception {
```

Comment:
can we ignore this test until it is stabilised please?

I can reproduce it 100% with:
```
mvn test -Dtest=hudson.PluginManagerTest#verifyUploadedPluginPermission
```

It passes in IntelliJ though

(have tried a bit but haven't figured it out)

Reply:
(although this test does pass for me on master weirdly although failing I think most of the time on CI)

Reply:
Tests pass for me locally with Java 11 on RHEL 8 with the command:

```
mvn clean -DforkCount=1C -Dtest=PluginManagerTest,DirectoryBrowserSupportTest,ComputerSetTest,ExecutorTest,SearchTest clean verify
```